packages/documentation/src/content/docs/integration/event-handlers.mdx

```diff
@@ -100,11 +120,27 @@ Example: An outgoing payment for $12 has been completed. $11.50 were sent. The A
 `}
 />
 
+Example: An outgoing payment supporting two-phase transfers for **$12** has been completed. **$11.50** were sent. The Account Servicing Entity keeps **$0.50** as fees.
```

Comment:
Hehe, I picked that up and fixed it. It was because of the `$`, and I think a combination of other phrases, but the `&` was definitely the trigger. 
I fixed it with `**&dollar;12**`

Reply:
The way it looks on my local now. @mkurapov 
<img width="677" alt="image" src="https://github.com/interledger/rafiki/assets/1992108/fcf0c02e-42b7-4bbc-9430-526b20fad913">

Reply:
@koekiebox Yes, thank you for updating it! This section looks good, but the section just [below](https://deploy-preview-2671--brilliant-pasca-3e80ec.netlify.app/integration/event-handlers/#outgoing_paymentcompleted) still has odd formatting

Reply:
Ah, I missed that one @mkurapov  ! It has something to do with the `for` being in front of the `$`.